io/net/virt-net/network_virtualization.py

```diff
@@ -390,6 +390,7 @@ def test_vnic_dlpar(self):
             except CmdError as details:
                 self.log.debug(str(details))
                 self.fail("dlpar operation did not complete")
+            time.sleep(30)
```

Comment:
There is a function `interfacewait()`. Let us use that instead of a hardcoded sleep value

Reply:
Also, we can enhance that function to see if IPAddr is set, pinging, etc.

Reply:
fixed.. more enhancement will be in avocado network utility